ydb/core/tx/schemeshard/schemeshard_path_describer.cpp

```diff
@@ -499,7 +499,7 @@ void TPathDescriber::DescribeTable(const TActorContext& ctx, TPathId pathId, TPa
 
         switch (childPath->PathType) {
         case NKikimrSchemeOp::EPathTypeTableIndex:
-            Self->DescribeTableIndex(childPathId, childName, returnConfig, false, *entry->AddTableIndexes());
+            Self->DescribeTableIndex(childPathId, childName, returnConfig, returnBoundaries, *entry->AddTableIndexes());
```

Comment:
This is the most problematic change.

We inherit `returnBoundaries` from the parent table's describe options and return indexImplTable boundaries even if the user is interested only in the main table boundaries. It might induce unnecessary message passing between actors in YDB especially since indexImplTables might have a very large number of partitions.

One way to solve this issue is to add a separate describe table [parameter](https://github.com/ydb-platform/ydb/blob/6b9001c607447a9e94cae8f4c3fbab768463019a/ydb/core/protos/flat_scheme_op.proto#L1834) to control the indexImplTable boundaries retrieval.

Reply:
Fixed in the latest [revision](https://github.com/ydb-platform/ydb/pull/7589/commits/33900bb676aeff4ce3702507a12590c901554f24)